doc-content/enterprise-only/project-data/select-branches-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='select-branches-proc']
+
+= Selecting branches
+You can switch between branches to test them and review their assets.
+
+.Procedure
+. Click on the current branch name menu and select the desired project branch.
++
+.Select a branch menu
+image::getting-started/change-branch.png[Selecting a branch menu]
+
+After selecting the branch, you will be redirected to that branch containing the project and all of the assets that you had defined for that branch.
```

Comment:
I think it's better to do "you are redirected" in this case. Future is appropriate in very limited cases.